src/databricks/labs/ucx/source_code/notebooks/sources.py

```diff
@@ -297,26 +319,6 @@ def name() -> str:
         return "notebook-linter"
 
 
-SUPPORTED_EXTENSION_LANGUAGES = {
```

Comment:
moved to base.by to avoid cyclic imports between cells.py, loaders.py, sources.py and files.py